packages/grid_client/src/modules/base.ts

```diff
@@ -135,8 +174,8 @@ class BaseModule {
     ext2 = await Promise.all(ext2.flat(1));
 
     if (this.backendStorage.type === BackendStorageType.tfkvstore) {
-      ext1 = ext1.filter(x => !!x);
-      ext2 = ext2.filter(x => !!x);
+      ext1 = ext1.flat(1).filter(x => !!x);
+      ext2 = ext2.flat(1).filter(x => !!x);
 
       await this.tfClient.applyAllExtrinsics(ext1);
       await this.tfClient.applyAllExtrinsics(ext2);
```

Comment:
let's apply these extrinsics together but the new ones first on the list then the old ones

Reply:
@AhmedHanafy725 What text should I add on the tooltip for explaining pass-through?

Reply:
> let's apply these extrinsics together but the new ones first on the list then the old ones

Merged both together

Reply:
Added tls pass through option